lib/config/validation.spec.ts

```diff
@@ -1225,6 +1225,30 @@ describe('config/validation', () => {
       expect(warnings).toHaveLength(0);
       expect(errors).toHaveLength(2);
     });
+
+    it('catches when * or ** is combined with others patterns in a regexOrGlob option', async () => {
+      const config = {
+        packageRules: [
+          {
+            matchRepositories: ['*', 'repo'],
```

Comment:
```suggestion
            matchRepositories: ['groupA/**', 'groupB/**'],
```
We should add a positive check that we allow multiple of such patterns.